infrastructure/main.tf

```diff
@@ -99,37 +114,37 @@ resource "azurerm_key_vault_secret" "error_notifications_username" {
 resource "azurerm_key_vault_secret" "POSTGRES-USER" {
   name         = "${var.component}-POSTGRES-USER"
   key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
-  value        = "${module.blob-router-db.user_name}"
+  value        = "${module.reform-blob-router-db.user_name}"
```

Comment:
should we define new resource ?

Reply:
as it is not live , we can directly start using new one?

Reply:
since blob router is not live, can we just bump the version in the original module and that's it? we don't care about any data in any environments atm now do we?

Reply:
will bumping the version destroy and recreate a new DB ?

Reply:
i sure hope so

Reply:
ok currently there is a lock on database which needs to be removed

Reply:
tried in perftest, getting error:
Error: Error waiting for deployment: Code="DeploymentFailed" Message="At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details." Details=[{"code":"Conflict","message":"{\r\n  \"status\": \"Failed\",\r\n  \"error\": {\r\n    \"code\": \"ResourceDeploymentFailure\",\r\n    \"message\": \"The resource operation completed with terminal provisioning state 'Failed'.\",\r\n    \"details\": [\r\n      {\r\n        \"code\": \"UnsupportedUpdate\",\r\n        \"message\": \"Cannot update Version for server\"\r\n      }\r\n    ]\r\n  }\r\n}"}]